VogtPayroll5/Payroll.cs

```diff
@@ -1,34 +1,90 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using System.IO;
 
 namespace VogtPayroll5
 {
     class Payroll
     {
+        //total number of hours worked by all employees, and total gross pay as well
+        private static int countHours;
+        private static decimal totalGrossPay;
+
         private List<Employee> _empList = new List<Employee>();
+
         public void Run()
         {
-            List<Employee> empList = new List<Employee>();
             PayrollConsoleReader payrollConsoleReader = new PayrollConsoleReader();
             Employee emp = new Employee();
             bool run = false;
 
             do
             {
+                Console.Clear();
                 emp = payrollConsoleReader.GetEmployeeInfo();
+                run = payrollConsoleReader.ReturnTrueOrFalse();
+
                 _empList.Add(emp);
             } while (run == true);
 
-            LoopThroughEmployeeInfo(emp.CalculateGrossPay(emp.HoursWorked, emp.HourlyPayRate));
+            LoopThroughEmployeeInfo(emp.CalculateGrossPay(emp.HoursWorked, emp.HourlyPayRate)+emp.CalculateOvertimePay(emp.HoursWorked, emp.HourlyPayRate));
+
+            DisplayTotalEmployeesHoursWorkedAndGrossPay();
+
+            WriteEmployeeListTotalsAndCountToFile(_empList);
+
+            File.Copy(@"C:\Users\Gabriel\source\repos\VogtPayroll5\EmployeeList.txt", @"C:\Users\Gabriel\source\repos\VogtPayroll5\EmployeeListBackup.txt");
+        }
+
+        public void DisplayTotalEmployeesHoursWorkedAndGrossPay()
+        {
+            Console.WriteLine();
+            Console.WriteLine($"Employee count: {_empList.Count}");
+            Console.WriteLine($"Employee total of hours worked {countHours}");
+            Console.WriteLine($"Employee total of gross pay {totalGrossPay:C2}");
+
         }
 
         public void LoopThroughEmployeeInfo(decimal grossPay)
         {
+            Console.Clear();
             foreach (var employee in _empList)
             {
+                Console.WriteLine();
+
                 employee.DisplayPayStatement(employee.EmpID, employee.EmpName, employee.HoursWorked, employee.HourlyPayRate, grossPay);
+
+                countHours += employee.HoursWorked;
+                totalGrossPay += grossPay;
+
+            }
+        }
+
+        public void WriteEmployeeListTotalsAndCountToFile(List<Employee> empList)
```

Comment:
I don't think the Payroll class should be responsible for writing to a file. It should know how to manage them. I.e. add them, delete, update, etc. Not how to print them.

Another class should be responsible for getting the employee list FROM the payroll manager, then looping through that employee list to write the file. But NOT the payroll manager class.